ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx

```diff
@@ -108,6 +111,7 @@ const LivePage = ({ pageData }: ComponentProps) => {
       <MetadataContainer
         title={pageTitle}
         lang={lang}
+        promoImage={promoImage}
```

Comment:
Can we use the existing `image`, `imageAltText`, `imageWidth` and `imageHeight` props to pass in the promo image? Would save us having to do the checks within the `Metadata` component.

Reply:
yes good idea!